dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -2272,7 +2272,7 @@ struct BundleHelper {
     prev_size_ = size_;
     prev_padding_ = padding_;
     gen_find_size(val, size_, padding_);
-    if ((size_ + padding_) > max_bundle_size_) {
+    if ((size_ + padding_) > (max_bundle_size_ - 100)) {
```

Comment:
> Message bundling does not currently handle DATA or DATA_FRAG submessages (they use a different path through the code).

Yes. I put this PR rather quickly and missed this change which I did more to "mark" where I needed to make the real change later. Changes from https://github.com/iguessthislldo/OpenDDS/pull/12 addresses this more properly. I will also update the title.

> If the bundling size needs to get reduced to accommodate security prefixes & suffixes, I'd prefer to see this done on a per-writer basis, based on whether those additional submessages would be added, rather than just a blanket reduction everywhere.

Absolutely agree. I started to work on that somewhat for DATA/FRAG side of things, but unfortunately this is just a quick fix.